cachalot/utils.py

```diff
@@ -105,18 +105,15 @@ def _find_subqueries_in_where(children):
             raise IsRawQuery
         elif child_class is NothingNode:
             pass
+        elif child_class is Subquery or child_class is Exists:
```

Comment:
Can you confirm Subquery is also an issue? Any way that I can reproduce this?

Reply:
I'm not sure about Subquery, The only test case I have currently is tied to our code base: 

```
from django.db.models import OuterRef
from apps.fms.models import Field
from django.db.models import Exists
from apps.fms.models import ProductApplied

year=2019
group_uuid='57ece8fc-5cba-11ea-a6bd-aaaaaaaa'

fields = Field.objects.filter(group__uuid=group_uuid)
fields = fields.filter(
    Exists(ProductApplied.objects.filter(field=OuterRef("pk"), harvest_year=year))
)
fields.all()
```
This will produce the error for us, I image you can do something simlar in what ever code your using.